library/core/src/pin.rs

```diff
@@ -364,7 +363,6 @@
 //! [`RefCell<T>`]: crate::cell::RefCell
```

Comment:
Your script missed this too.

Reply:
Well, unless `RefCell` is imported, it won't be auto-resolved, and my script isn't _that_ smart :)

Reply:
Oh oops 😆 yup this can't be removed.